lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -263,6 +274,85 @@ def self.get_github_release_tag_names(repo_name)
           :other
         end
       end
+
+      private_class_method def self.get_type_of_bump_from_types_of_change(change_types)
+        if change_types.include?("breaking")
+          :major
+        elsif change_types.include?("feat")
+          :minor
+        else
+          :patch
+        end
+      end
+
+      private_class_method def self.increase_version(current_version, type_of_bump, snapshot)
+        version_split = current_version.split('.')
+        UI.user_error("Invalid version number: #{current_version}. Expected 3 numbers separated by '.'") if version_split.size != 3
+
+        major = version_split[0]
+        minor = version_split[1]
+        patch = version_split[2]
+
+        case type_of_bump
+        when :major
+          next_version = "#{major.to_i + 1}.0.0"
+        when :minor
+          next_version = "#{major}.#{minor.to_i + 1}.0"
+        else
+          next_version = "#{major}.#{minor}.#{patch.to_i + 1}"
+        end
+
+        if snapshot
+          "#{next_version}-SNAPSHOT"
+        else
+          next_version
+        end
+      end
+
+      private_class_method def self.get_pr_resp_items_for_sha(sha, github_token, org, rate_limit_sleep, repo_name)
+        return @pr_resp_items_by_sha[sha] if @pr_resp_items_by_sha.include?(sha)
+
+        if rate_limit_sleep > 0
+          UI.message("Sleeping #{rate_limit_sleep} second(s) to avoid rate limit 🐌")
+          sleep(rate_limit_sleep)
+        end
+
+        # Get pull request associate with commit message
+        pr_resp = Actions::GithubApiAction.run(server_url: 'https://api.github.com',
+                                               path: "/search/issues?q=repo:#{org}/#{repo_name}+is:pr+base:main+SHA:#{sha}",
+                                               http_method: 'GET',
+                                               body: {},
+                                               api_token: github_token)
+        body = JSON.parse(pr_resp[:body])
+        items = body["items"]
+        @pr_resp_items_by_sha[sha] ||= items
```

Comment:
Hmm I guess this could be a simple assignment, since we have already checked before that the map doesn't contain the value. NABD though
```suggestion
        @pr_resp_items_by_sha[sha] = items
```